src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs

```diff
@@ -1677,7 +1677,15 @@ private EventBuilder DefineEventNoLock(string name, EventAttributes attributes,
         #region Create Type
 
         [return: DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
-        public TypeInfo? CreateTypeInfo()
+        public TypeInfo CreateTypeInfo()
+        {
+            TypeInfo? typeInfo = CreateTypeInfoImpl();
+            Debug.Assert(typeInfo != null);
+            return typeInfo;
+        }
+
+        [return: DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
+        internal TypeInfo? CreateTypeInfoImpl()
```

Comment:
CreateTypeInfoImpl is identical to CreateTypeImpl. Could you please delete CreateTypeImpl and make CreateTypeInfo to call CreateTypeImpl instead?

Reply:
Changed to only have 1 internal method with a signature of `internal TypeInfo? CreateTypeInfoImpl()`.